src/components/vis/Map/index.jsx

```diff
@@ -0,0 +1,235 @@
+import React from 'react';
+import L from 'leaflet/dist/leaflet';
+import 'leaflet/dist/leaflet.css';
+import isEqual from 'lodash/isEqual';
+import Spinner from 'components/ui/Spinner';
+import './style.scss';
+
+const MAP_CONFIG = {
+  zoom: 2,
+  minZoom: 2,
+  latLng: {
+    lat: 30,
+    lng: -120
+  },
+  zoomControl: true
+};
+
+class Map extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      sidebarOpen: props.sidebar.open,
```

Comment:
Don't you think that would be better to store only the sidebar in the state? Imagine that you have 5 params inside sidebar, it would be overwhelming to split them in different vars